src/Features/Core/Portable/FeaturesResources.resx

```diff
@@ -1753,4 +1753,7 @@ This version used in: {2}</value>
   <data name="Value_colon" xml:space="preserve">
     <value>Value:</value>
   </data>
+  <data name="Implement_remaining_members_of_interface_explicitly" xml:space="preserve">
+    <value>Implement remaining members of interface explicitly</value>
```

Comment:
I'd simplify to `Implement remaining members explicitly` and maybe rename the existing one to `Implement all members explicitly`